python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Abkhasians_abh100-M-LP6005442-DNA_D02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Abkhasians_abh100-M-LP6005442-DNA_D02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Abkhasians_abh100-M-LP6005442-DNA_D02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Abkhasians_abh107-M-LP6005442-DNA_C02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Abkhasians_abh107-M-LP6005442-DNA_C02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Abkhasians_abh107-M-LP6005442-DNA_C02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Adygei_HGDP01401-F-LP6005441-DNA_B01 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Adygei_HGDP01401-F-LP6005441-DNA_B01 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Adygei_HGDP01401-F-LP6005441-DNA_B01
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Adygei_HGDP01402-M-LP6005441-DNA_A01 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Adygei_HGDP01402-M-LP6005441-DNA_A01 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Adygei_HGDP01402-M-LP6005441-DNA_A01
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Aleut_Ale20-F-LP6005443-DNA_H02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Aleut_Ale20-F-LP6005443-DNA_H02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Aleut_Ale20-F-LP6005443-DNA_H02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Aleut_Ale22-F-LP6005443-DNA_A03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Aleut_Ale22-F-LP6005443-DNA_A03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Aleut_Ale22-F-LP6005443-DNA_A03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Altaian_altai363p-M-LP6005442-DNA_F02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Altaian_altai363p-M-LP6005442-DNA_F02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Altaian_altai363p-M-LP6005442-DNA_F02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Ami_NA13607-M-LP6005442-DNA_C07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Ami_NA13607-M-LP6005442-DNA_C07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Ami_NA13607-M-LP6005442-DNA_C07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Ami_NA13616-M-LP6005443-DNA_G05 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Ami_NA13616-M-LP6005443-DNA_G05 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Ami_NA13616-M-LP6005443-DNA_G05
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Armenians_armenia293-M-LP6005442-DNA_G02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Armenians_armenia293-M-LP6005442-DNA_G02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Armenians_armenia293-M-LP6005442-DNA_G02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Australia_ECCAC_IHW9118-F-LP6005442-DNA_A09 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Australia_ECCAC_IHW9118-F-LP6005442-DNA_A09 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Australia_ECCAC_IHW9118-F-LP6005442-DNA_A09
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Australia_ECCAC_IHW9193-M-LP6005442-DNA_B09 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Australia_ECCAC_IHW9193-M-LP6005442-DNA_B09 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Australia_ECCAC_IHW9193-M-LP6005442-DNA_B09
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Balochi_HGDP00058-M-LP6005441-DNA_C01 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Balochi_HGDP00058-M-LP6005441-DNA_C01 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Balochi_HGDP00058-M-LP6005441-DNA_C01
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Balochi_HGDP00090-M-LP6005441-DNA_D01 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Balochi_HGDP00090-M-LP6005441-DNA_D01 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Balochi_HGDP00090-M-LP6005441-DNA_D01
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_BantuKenya_HGDP01414-F-LP6005441-DNA_B02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_BantuKenya_HGDP01414-F-LP6005441-DNA_B02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_BantuKenya_HGDP01414-F-LP6005441-DNA_B02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_BantuKenya_HGDP01417-M-LP6005443-DNA_A01 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_BantuKenya_HGDP01417-M-LP6005443-DNA_A01 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_BantuKenya_HGDP01417-M-LP6005443-DNA_A01
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Bantu_SE_Herero_HGDP01028-M-LP6005443-DNA_E02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Bantu_SE_Herero_HGDP01028-M-LP6005443-DNA_E02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Bantu_SE_Herero_HGDP01028-M-LP6005443-DNA_E02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Bantu_SE_Herero_HGDP01035-M-LP6005441-DNA_F01 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Bantu_SE_Herero_HGDP01035-M-LP6005441-DNA_F01 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Bantu_SE_Herero_HGDP01035-M-LP6005441-DNA_F01
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Bantu_SE_Tswana_HGDP01030-M-LP6005443-DNA_G02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Bantu_SE_Tswana_HGDP01030-M-LP6005443-DNA_G02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Bantu_SE_Tswana_HGDP01030-M-LP6005443-DNA_G02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Bantu_SE_Tswana_HGDP01034-M-LP6005443-DNA_F02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Bantu_SE_Tswana_HGDP01034-M-LP6005443-DNA_F02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Bantu_SE_Tswana_HGDP01034-M-LP6005443-DNA_F02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Basque_HGDP01364-M-LP6005441-DNA_C02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Basque_HGDP01364-M-LP6005441-DNA_C02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Basque_HGDP01364-M-LP6005441-DNA_C02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Basque_HGDP01365-F-LP6005441-DNA_D02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Basque_HGDP01365-F-LP6005441-DNA_D02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Basque_HGDP01365-F-LP6005441-DNA_D02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_BEB_Bengali_HG03006-M-LP6005442-DNA_G09 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_BEB_Bengali_HG03006-M-LP6005442-DNA_G09 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_BEB_Bengali_HG03006-M-LP6005442-DNA_G09
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_BEB_Bengali_HG03007-F-LP6005442-DNA_H09 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_BEB_Bengali_HG03007-F-LP6005442-DNA_H09 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_BEB_Bengali_HG03007-F-LP6005442-DNA_H09
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Bedouin_HGDP00616-M-LP6005441-DNA_E02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Bedouin_HGDP00616-M-LP6005441-DNA_E02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Bedouin_HGDP00616-M-LP6005441-DNA_E02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Bedouin_HGDP00650-F-LP6005441-DNA_F02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Bedouin_HGDP00650-F-LP6005441-DNA_F02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Bedouin_HGDP00650-F-LP6005441-DNA_F02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_BiakaPygmy_HGDP00457-M-LP6005441-DNA_G02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_BiakaPygmy_HGDP00457-M-LP6005441-DNA_G02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_BiakaPygmy_HGDP00457-M-LP6005441-DNA_G02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_BiakaPygmy_HGDP00461-M-LP6005441-DNA_H02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_BiakaPygmy_HGDP00461-M-LP6005441-DNA_H02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_BiakaPygmy_HGDP00461-M-LP6005441-DNA_H02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Bougainville_HGDP00656-F-LP6005441-DNA_A03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Bougainville_HGDP00656-F-LP6005441-DNA_A03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Bougainville_HGDP00656-F-LP6005441-DNA_A03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Bougainville_HGDP00660-F-LP6005441-DNA_B03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Bougainville_HGDP00660-F-LP6005441-DNA_B03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Bougainville_HGDP00660-F-LP6005441-DNA_B03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Brahui_HGDP00019-M-LP6005441-DNA_C03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Brahui_HGDP00019-M-LP6005441-DNA_C03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Brahui_HGDP00019-M-LP6005441-DNA_C03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Brahui_HGDP00027-M-LP6005441-DNA_D03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Brahui_HGDP00027-M-LP6005441-DNA_D03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Brahui_HGDP00027-M-LP6005441-DNA_D03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Bulgarians_BulgarianB4-M-LP6005442-DNA_A03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Bulgarians_BulgarianB4-M-LP6005442-DNA_A03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Bulgarians_BulgarianB4-M-LP6005442-DNA_A03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Bulgarians_BulgarianC1-M-LP6005442-DNA_B03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Bulgarians_BulgarianC1-M-LP6005442-DNA_B03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Bulgarians_BulgarianC1-M-LP6005442-DNA_B03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Burusho_HGDP00338-F-LP6005441-DNA_F03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Burusho_HGDP00338-F-LP6005441-DNA_F03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Burusho_HGDP00338-F-LP6005441-DNA_F03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Burusho_HGDP00428-M-LP6005441-DNA_E03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Burusho_HGDP00428-M-LP6005441-DNA_E03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Burusho_HGDP00428-M-LP6005441-DNA_E03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Cambodian_HGDP00713-F-LP6005441-DNA_H03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Cambodian_HGDP00713-F-LP6005441-DNA_H03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Cambodian_HGDP00713-F-LP6005441-DNA_H03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Cambodian_HGDP00717-M-LP6005441-DNA_G03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Cambodian_HGDP00717-M-LP6005441-DNA_G03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Cambodian_HGDP00717-M-LP6005441-DNA_G03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Chechens_ch113-M-LP6005442-DNA_D03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Chechens_ch113-M-LP6005442-DNA_D03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Chechens_ch113-M-LP6005442-DNA_D03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Chipewyan_NAD59-M-LP6005443-DNA_F05 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Chipewyan_NAD59-M-LP6005443-DNA_F05 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Chipewyan_NAD59-M-LP6005443-DNA_F05
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Chipewyan_NAD93-F-LP6005442-DNA_H06 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Chipewyan_NAD93-F-LP6005442-DNA_H06 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Chipewyan_NAD93-F-LP6005442-DNA_H06
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Chukchi_Sir19-M-LP6005443-DNA_C03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Chukchi_Sir19-M-LP6005443-DNA_C03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Chukchi_Sir19-M-LP6005443-DNA_C03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Chukchi_Sir26-M-LP6005443-DNA_B03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Chukchi_Sir26-M-LP6005443-DNA_B03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Chukchi_Sir26-M-LP6005443-DNA_B03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Cree_Cree5-M-LP6005442-DNA_A07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Cree_Cree5-M-LP6005442-DNA_A07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Cree_Cree5-M-LP6005442-DNA_A07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Cree_Cree6-F-LP6005442-DNA_B07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Cree_Cree6-F-LP6005442-DNA_B07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Cree_Cree6-F-LP6005442-DNA_B07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Czechoslovakian_NA15724-M-LP6005442-DNA_E07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Czechoslovakian_NA15724-M-LP6005442-DNA_E07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Czechoslovakian_NA15724-M-LP6005442-DNA_E07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Czechoslovakian_NA15728-M-LP6005443-DNA_H05 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Czechoslovakian_NA15728-M-LP6005443-DNA_H05 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Czechoslovakian_NA15728-M-LP6005443-DNA_H05
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Dai_HGDP01312-M-LP6005443-DNA_B01 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Dai_HGDP01312-M-LP6005443-DNA_B01 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Dai_HGDP01312-M-LP6005443-DNA_B01
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Dai_HGDP01315-F-LP6005441-DNA_D04 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Dai_HGDP01315-F-LP6005441-DNA_D04 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Dai_HGDP01315-F-LP6005441-DNA_D04
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Daur_HGDP01215-F-LP6005441-DNA_F04 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Daur_HGDP01215-F-LP6005441-DNA_F04 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Daur_HGDP01215-F-LP6005441-DNA_F04
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Daur_HGDP01217-M-LP6005443-DNA_C01 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Daur_HGDP01217-M-LP6005443-DNA_C01 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Daur_HGDP01217-M-LP6005443-DNA_C01
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Dinka_DNK11-M-LP6005443-DNA_B09 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Dinka_DNK11-M-LP6005443-DNA_B09 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Dinka_DNK11-M-LP6005443-DNA_B09
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Dinka_DNK5-M-LP6005443-DNA_H08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Dinka_DNK5-M-LP6005443-DNA_H08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Dinka_DNK5-M-LP6005443-DNA_H08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Druze_HGDP00569-F-LP6005443-DNA_D01 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Druze_HGDP00569-F-LP6005443-DNA_D01 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Druze_HGDP00569-F-LP6005443-DNA_D01
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Druze_HGDP00597-M-LP6005441-DNA_G04 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Druze_HGDP00597-M-LP6005441-DNA_G04 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Druze_HGDP00597-M-LP6005441-DNA_G04
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Eskimo_Chaplino_Esk29-M-LP6005443-DNA_D03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Eskimo_Chaplino_Esk29-M-LP6005443-DNA_D03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Eskimo_Chaplino_Esk29-M-LP6005443-DNA_D03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Eskimo_Naukan_Nesk22-F-LP6005443-DNA_F03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Eskimo_Naukan_Nesk22-F-LP6005443-DNA_F03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Eskimo_Naukan_Nesk22-F-LP6005443-DNA_F03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Eskimo_Naukan_Nesk25-F-LP6005443-DNA_G03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Eskimo_Naukan_Nesk25-F-LP6005443-DNA_G03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Eskimo_Naukan_Nesk25-F-LP6005443-DNA_G03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Eskimo_Sireniki_Sir40-F-LP6005443-DNA_H03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Eskimo_Sireniki_Sir40-F-LP6005443-DNA_H03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Eskimo_Sireniki_Sir40-F-LP6005443-DNA_H03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_ESN_Esan_HG02943-F-LP6005442-DNA_B10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_ESN_Esan_HG02943-F-LP6005442-DNA_B10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_ESN_Esan_HG02943-F-LP6005442-DNA_B10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_ESN_Esan_HG03100-M-LP6005442-DNA_A10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_ESN_Esan_HG03100-M-LP6005442-DNA_A10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_ESN_Esan_HG03100-M-LP6005442-DNA_A10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Estonian_Est375-M-LP6005442-DNA_H03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Estonian_Est375-M-LP6005442-DNA_H03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Estonian_Est375-M-LP6005442-DNA_H03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Estonian_Est400-M-LP6005442-DNA_G03 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Estonian_Est400-M-LP6005442-DNA_G03 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Estonian_Est400-M-LP6005442-DNA_G03
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Even_Nlk18-M-LP6005443-DNA_C04 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Even_Nlk18-M-LP6005443-DNA_C04 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Even_Nlk18-M-LP6005443-DNA_C04
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Even_Nlk3-F-LP6005443-DNA_B04 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Even_Nlk3-F-LP6005443-DNA_B04 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Even_Nlk3-F-LP6005443-DNA_B04
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_FIN_Finland_HG00174-F-LP6005442-DNA_D10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_FIN_Finland_HG00174-F-LP6005442-DNA_D10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_FIN_Finland_HG00174-F-LP6005442-DNA_D10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_FIN_Finland_HG00190-M-LP6005442-DNA_C10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_FIN_Finland_HG00190-M-LP6005442-DNA_C10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_FIN_Finland_HG00190-M-LP6005442-DNA_C10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_French_HGDP00526-F-LP6005441-DNA_B05 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_French_HGDP00526-F-LP6005441-DNA_B05 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_French_HGDP00526-F-LP6005441-DNA_B05
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_French_HGDP00530-M-LP6005441-DNA_A05 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_French_HGDP00530-M-LP6005441-DNA_A05 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_French_HGDP00530-M-LP6005441-DNA_A05
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_GBR_Kent_HG00126-M-LP6005442-DNA_E10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_GBR_Kent_HG00126-M-LP6005442-DNA_E10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_GBR_Kent_HG00126-M-LP6005442-DNA_E10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_GBR_Kent_HG00128-F-LP6005442-DNA_F10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_GBR_Kent_HG00128-F-LP6005442-DNA_F10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_GBR_Kent_HG00128-F-LP6005442-DNA_F10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Georgians_Megrels_mg27-M-LP6005442-DNA_B04 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Georgians_Megrels_mg27-M-LP6005442-DNA_B04 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Georgians_Megrels_mg27-M-LP6005442-DNA_B04
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Georgians_Megrels_mg31-M-LP6005442-DNA_A04 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Georgians_Megrels_mg31-M-LP6005442-DNA_A04 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Georgians_Megrels_mg31-M-LP6005442-DNA_A04
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Greek_NA17374-M-LP6005443-DNA_A06 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Greek_NA17374-M-LP6005443-DNA_A06 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Greek_NA17374-M-LP6005443-DNA_A06
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Greek_NA17377-M-LP6005442-DNA_G07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Greek_NA17377-M-LP6005442-DNA_G07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Greek_NA17377-M-LP6005442-DNA_G07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_GWD_Gambia_HG02464-M-LP6005442-DNA_G10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_GWD_Gambia_HG02464-M-LP6005442-DNA_G10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_GWD_Gambia_HG02464-M-LP6005442-DNA_G10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_GWD_Gambia_HG02574-F-LP6005442-DNA_H10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_GWD_Gambia_HG02574-F-LP6005442-DNA_H10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_GWD_Gambia_HG02574-F-LP6005442-DNA_H10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Han_HGDP00783-F-LP6005441-DNA_D05 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Han_HGDP00783-F-LP6005441-DNA_D05 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Han_HGDP00783-F-LP6005441-DNA_D05
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Han_HGDP00785-M-LP6005441-DNA_C05 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Han_HGDP00785-M-LP6005441-DNA_C05 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Han_HGDP00785-M-LP6005441-DNA_C05
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Hazara_HGDP00124-M-LP6005441-DNA_F05 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Hazara_HGDP00124-M-LP6005441-DNA_F05 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Hazara_HGDP00124-M-LP6005441-DNA_F05
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Hazara_HGDP00125-M-LP6005441-DNA_E05 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Hazara_HGDP00125-M-LP6005441-DNA_E05 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Hazara_HGDP00125-M-LP6005441-DNA_E05
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Hezhen_HGDP01240-M-LP6005441-DNA_G05 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Hezhen_HGDP01240-M-LP6005441-DNA_G05 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Hezhen_HGDP01240-M-LP6005441-DNA_G05
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Hezhen_HGDP01242-F-LP6005441-DNA_H05 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Hezhen_HGDP01242-F-LP6005441-DNA_H05 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Hezhen_HGDP01242-F-LP6005441-DNA_H05
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Hungarian_NA15202-M-LP6005442-DNA_A08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Hungarian_NA15202-M-LP6005442-DNA_A08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Hungarian_NA15202-M-LP6005442-DNA_A08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Hungarian_NA15203-F-LP6005442-DNA_B08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Hungarian_NA15203-F-LP6005442-DNA_B08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Hungarian_NA15203-F-LP6005442-DNA_B08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_IBS_Castilla_la_Mancha_HG01503-M-LP6005442-DNA_A11 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_IBS_Castilla_la_Mancha_HG01503-M-LP6005442-DNA_A11 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_IBS_Castilla_la_Mancha_HG01503-M-LP6005442-DNA_A11
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_IBS_Castilla_la_Mancha_HG01504-F-LP6005442-DNA_B11 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_IBS_Castilla_la_Mancha_HG01504-F-LP6005442-DNA_B11 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_IBS_Castilla_la_Mancha_HG01504-F-LP6005442-DNA_B11
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Icelanders_NA15761-F-LP6005442-DNA_D08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Icelanders_NA15761-F-LP6005442-DNA_D08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Icelanders_NA15761-F-LP6005442-DNA_D08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Icelanders_NA15763-F-LP6005443-DNA_B06 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Icelanders_NA15763-F-LP6005443-DNA_B06 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Icelanders_NA15763-F-LP6005443-DNA_B06
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Iranians_iran11-F-LP6005443-DNA_B10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Iranians_iran11-F-LP6005443-DNA_B10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Iranians_iran11-F-LP6005443-DNA_B10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Iranians_iran17-M-LP6005442-DNA_C04 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Iranians_iran17-M-LP6005442-DNA_C04 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Iranians_iran17-M-LP6005442-DNA_C04
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Italian_HGDP01153-M-LP6005441-DNA_A06 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Italian_HGDP01153-M-LP6005441-DNA_A06 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Italian_HGDP01153-M-LP6005441-DNA_A06
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Italian_HGDP01172-F-LP6005441-DNA_B06 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Italian_HGDP01172-F-LP6005441-DNA_B06 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Italian_HGDP01172-F-LP6005441-DNA_B06
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Itelmen_Kor82-F-LP6005443-DNA_D04 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Itelmen_Kor82-F-LP6005443-DNA_D04 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Itelmen_Kor82-F-LP6005443-DNA_D04
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Japanese_HGDP00749-M-LP6005441-DNA_C06 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Japanese_HGDP00749-M-LP6005441-DNA_C06 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Japanese_HGDP00749-M-LP6005441-DNA_C06
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Japanese_HGDP00773-F-LP6005441-DNA_D06 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Japanese_HGDP00773-F-LP6005441-DNA_D06 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Japanese_HGDP00773-F-LP6005441-DNA_D06
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Jordanians_Jordan214-M-LP6005442-DNA_E04 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Jordanians_Jordan214-M-LP6005442-DNA_E04 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Jordanians_Jordan214-M-LP6005442-DNA_E04
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Jordanians_Jordan603-M-LP6005442-DNA_F04 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Jordanians_Jordan603-M-LP6005442-DNA_F04 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Jordanians_Jordan603-M-LP6005442-DNA_F04
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Kalash_HGDP00286-F-LP6005441-DNA_F06 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Kalash_HGDP00286-F-LP6005441-DNA_F06 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Kalash_HGDP00286-F-LP6005441-DNA_F06
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Kalash_HGDP00328-M-LP6005441-DNA_E06 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Kalash_HGDP00328-M-LP6005441-DNA_E06 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Kalash_HGDP00328-M-LP6005441-DNA_E06
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Karitiana_HGDP01012-M-LP6005441-DNA_G06 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Karitiana_HGDP01012-M-LP6005441-DNA_G06 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Karitiana_HGDP01012-M-LP6005441-DNA_G06
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Karitiana_HGDP01018-F-LP6005441-DNA_H06 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Karitiana_HGDP01018-F-LP6005441-DNA_H06 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Karitiana_HGDP01018-F-LP6005441-DNA_H06
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_KHV_Kinh_HG01600-F-LP6005442-DNA_D11 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_KHV_Kinh_HG01600-F-LP6005442-DNA_D11 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_KHV_Kinh_HG01600-F-LP6005442-DNA_D11
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_KHV_Kinh_HG01846-M-LP6005442-DNA_C11 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_KHV_Kinh_HG01846-M-LP6005442-DNA_C11 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_KHV_Kinh_HG01846-M-LP6005442-DNA_C11
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Koreans_NA00726-F-LP6005443-DNA_C06 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Koreans_NA00726-F-LP6005443-DNA_C06 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Koreans_NA00726-F-LP6005443-DNA_C06
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Koreans_ND19394-M-LP6005443-DNA_D06 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Koreans_ND19394-M-LP6005443-DNA_D06 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Koreans_ND19394-M-LP6005443-DNA_D06
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Kusunda_K_15-M-LP6005443-DNA_D09 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Kusunda_K_15-M-LP6005443-DNA_D09 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Kusunda_K_15-M-LP6005443-DNA_D09
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Kusunda_K_2-M-LP6005443-DNA_C09 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Kusunda_K_2-M-LP6005443-DNA_C09 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Kusunda_K_2-M-LP6005443-DNA_C09
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Lahu_HGDP01320-M-LP6005443-DNA_E01 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Lahu_HGDP01320-M-LP6005443-DNA_E01 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Lahu_HGDP01320-M-LP6005443-DNA_E01
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Lahu_HGDP01323-F-LP6005441-DNA_B07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Lahu_HGDP01323-F-LP6005441-DNA_B07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Lahu_HGDP01323-F-LP6005441-DNA_B07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Lezgins_lez42-M-LP6005442-DNA_G04 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Lezgins_lez42-M-LP6005442-DNA_G04 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Lezgins_lez42-M-LP6005442-DNA_G04
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Lezgins_lez49-M-LP6005442-DNA_H04 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Lezgins_lez49-M-LP6005442-DNA_H04 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Lezgins_lez49-M-LP6005442-DNA_H04
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Luo_Ayodo_502C-F-LP6005442-DNA_F09 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Luo_Ayodo_502C-F-LP6005442-DNA_F09 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Luo_Ayodo_502C-F-LP6005442-DNA_F09
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_LWK_Luhya_NA19023-F-LP6005442-DNA_F11 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_LWK_Luhya_NA19023-F-LP6005442-DNA_F11 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_LWK_Luhya_NA19023-F-LP6005442-DNA_F11
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_LWK_Luhya_NA19044-M-LP6005442-DNA_E11 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_LWK_Luhya_NA19044-M-LP6005442-DNA_E11 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_LWK_Luhya_NA19044-M-LP6005442-DNA_E11
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Makrani_HGDP00157-F-LP6005441-DNA_D07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Makrani_HGDP00157-F-LP6005441-DNA_D07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Makrani_HGDP00157-F-LP6005441-DNA_D07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Makrani_HGDP00160-M-LP6005441-DNA_C07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Makrani_HGDP00160-M-LP6005441-DNA_C07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Makrani_HGDP00160-M-LP6005441-DNA_C07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Mandenka_HGDP00915-F-LP6005441-DNA_F07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Mandenka_HGDP00915-F-LP6005441-DNA_F07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Mandenka_HGDP00915-F-LP6005441-DNA_F07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Mandenka_HGDP01199-M-LP6005441-DNA_E07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Mandenka_HGDP01199-M-LP6005441-DNA_E07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Mandenka_HGDP01199-M-LP6005441-DNA_E07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Mansi_Mansi41-F-LP6005443-DNA_G04 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Mansi_Mansi41-F-LP6005443-DNA_G04 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Mansi_Mansi41-F-LP6005443-DNA_G04
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Mansi_Mansi79-M-LP6005443-DNA_F04 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Mansi_Mansi79-M-LP6005443-DNA_F04 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Mansi_Mansi79-M-LP6005443-DNA_F04
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Maya_HGDP00855-F-LP6005441-DNA_G07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Maya_HGDP00855-F-LP6005441-DNA_G07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Maya_HGDP00855-F-LP6005441-DNA_G07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Maya_HGDP00857-F-LP6005441-DNA_H07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Maya_HGDP00857-F-LP6005441-DNA_H07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Maya_HGDP00857-F-LP6005441-DNA_H07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_MbutiPygmy_HGDP00449-M-LP6005441-DNA_A08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_MbutiPygmy_HGDP00449-M-LP6005441-DNA_A08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_MbutiPygmy_HGDP00449-M-LP6005441-DNA_A08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_MbutiPygmy_HGDP00476-F-LP6005441-DNA_B08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_MbutiPygmy_HGDP00476-F-LP6005441-DNA_B08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_MbutiPygmy_HGDP00476-F-LP6005441-DNA_B08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Miao_HGDP01191-M-LP6005441-DNA_C08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Miao_HGDP01191-M-LP6005441-DNA_C08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Miao_HGDP01191-M-LP6005441-DNA_C08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Miao_HGDP01198-F-LP6005441-DNA_D08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Miao_HGDP01198-F-LP6005441-DNA_D08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Miao_HGDP01198-F-LP6005441-DNA_D08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Mixe_mixe0002-F-LP6005443-DNA_E11 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Mixe_mixe0002-F-LP6005443-DNA_E11 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Mixe_mixe0002-F-LP6005443-DNA_E11
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Mixe_mixe0042-F-LP6005443-DNA_F11 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Mixe_mixe0042-F-LP6005443-DNA_F11 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Mixe_mixe0042-F-LP6005443-DNA_F11
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Mixtec_Mixa0099-M-LP6005443-DNA_G11 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Mixtec_Mixa0099-M-LP6005443-DNA_G11 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Mixtec_Mixa0099-M-LP6005443-DNA_G11
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Mixtec_mixa0105-F-LP6005443-DNA_H11 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Mixtec_mixa0105-F-LP6005443-DNA_H11 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Mixtec_mixa0105-F-LP6005443-DNA_H11
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_MKK_Masai_NA21490-M-LP6005443-DNA_F06 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_MKK_Masai_NA21490-M-LP6005443-DNA_F06 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_MKK_Masai_NA21490-M-LP6005443-DNA_F06
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_MKK_Masai_NA21581-M-LP6005443-DNA_E06 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_MKK_Masai_NA21581-M-LP6005443-DNA_E06 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_MKK_Masai_NA21581-M-LP6005443-DNA_E06
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Mongola_HGDP01223-F-LP6005441-DNA_F08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Mongola_HGDP01223-F-LP6005441-DNA_F08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Mongola_HGDP01223-F-LP6005441-DNA_F08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Mongola_HGDP01228-M-LP6005441-DNA_E08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Mongola_HGDP01228-M-LP6005441-DNA_E08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Mongola_HGDP01228-M-LP6005441-DNA_E08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Mozabite_HGDP01253-M-LP6005441-DNA_G08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Mozabite_HGDP01253-M-LP6005441-DNA_G08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Mozabite_HGDP01253-M-LP6005441-DNA_G08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Mozabite_HGDP01274-F-LP6005441-DNA_H08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Mozabite_HGDP01274-F-LP6005441-DNA_H08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Mozabite_HGDP01274-F-LP6005441-DNA_H08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_MSL_Mende_HG03078-M-LP6005442-DNA_G11 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_MSL_Mende_HG03078-M-LP6005442-DNA_G11 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_MSL_Mende_HG03078-M-LP6005442-DNA_G11
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_MSL_Mende_HG03085-F-LP6005442-DNA_H11 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_MSL_Mende_HG03085-F-LP6005442-DNA_H11 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_MSL_Mende_HG03085-F-LP6005442-DNA_H11
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Naxi_HGDP01338-M-LP6005443-DNA_E09 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Naxi_HGDP01338-M-LP6005443-DNA_E09 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Naxi_HGDP01338-M-LP6005443-DNA_E09
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Naxi_HGDP01344-M-LP6005441-DNA_A09 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Naxi_HGDP01344-M-LP6005441-DNA_A09 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Naxi_HGDP01344-M-LP6005441-DNA_A09
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Naxi_HGDP01345-F-LP6005441-DNA_B09 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Naxi_HGDP01345-F-LP6005441-DNA_B09 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Naxi_HGDP01345-F-LP6005441-DNA_B09
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_NorthOssetians_NorthOssetia12-M-LP6005443-DNA_E10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_NorthOssetians_NorthOssetia12-M-LP6005443-DNA_E10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_NorthOssetians_NorthOssetia12-M-LP6005443-DNA_E10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_NorthOssetians_NorthOssetia5-M-LP6005443-DNA_F10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_NorthOssetians_NorthOssetia5-M-LP6005443-DNA_F10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_NorthOssetians_NorthOssetia5-M-LP6005443-DNA_F10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Orcadian_HGDP00796-F-LP6005441-DNA_D09 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Orcadian_HGDP00796-F-LP6005441-DNA_D09 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Orcadian_HGDP00796-F-LP6005441-DNA_D09
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Orcadian_HGDP00798-M-LP6005441-DNA_C09 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Orcadian_HGDP00798-M-LP6005441-DNA_C09 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Orcadian_HGDP00798-M-LP6005441-DNA_C09
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Oroqen_HGDP01203-M-LP6005441-DNA_E09 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Oroqen_HGDP01203-M-LP6005441-DNA_E09 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Oroqen_HGDP01203-M-LP6005441-DNA_E09
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Oroqen_HGDP01211-F-LP6005441-DNA_F09 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Oroqen_HGDP01211-F-LP6005441-DNA_F09 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Oroqen_HGDP01211-F-LP6005441-DNA_F09
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Palestinian_HGDP00725-M-LP6005441-DNA_G09 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Palestinian_HGDP00725-M-LP6005441-DNA_G09 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Palestinian_HGDP00725-M-LP6005441-DNA_G09
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Palestinian_HGDP00737-F-LP6005441-DNA_H09 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Palestinian_HGDP00737-F-LP6005441-DNA_H09 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Palestinian_HGDP00737-F-LP6005441-DNA_H09
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Papuan_HGDP00540-M-LP6005443-DNA_H07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Papuan_HGDP00540-M-LP6005443-DNA_H07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Papuan_HGDP00540-M-LP6005443-DNA_H07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Papuan_HGDP00541-M-LP6005441-DNA_A10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Papuan_HGDP00541-M-LP6005441-DNA_A10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Papuan_HGDP00541-M-LP6005441-DNA_A10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Papuan_HGDP00543-M-LP6005443-DNA_E07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Papuan_HGDP00543-M-LP6005443-DNA_E07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Papuan_HGDP00543-M-LP6005443-DNA_E07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Papuan_HGDP00545-M-LP6005443-DNA_D07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Papuan_HGDP00545-M-LP6005443-DNA_D07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Papuan_HGDP00545-M-LP6005443-DNA_D07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Papuan_HGDP00547-M-LP6005443-DNA_C08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Papuan_HGDP00547-M-LP6005443-DNA_C08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Papuan_HGDP00547-M-LP6005443-DNA_C08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Papuan_HGDP00548-M-LP6005443-DNA_F07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Papuan_HGDP00548-M-LP6005443-DNA_F07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Papuan_HGDP00548-M-LP6005443-DNA_F07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Papuan_HGDP00549-M-LP6005443-DNA_A08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Papuan_HGDP00549-M-LP6005443-DNA_A08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Papuan_HGDP00549-M-LP6005443-DNA_A08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Papuan_HGDP00550-F-LP6005441-DNA_B10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Papuan_HGDP00550-F-LP6005441-DNA_B10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Papuan_HGDP00550-F-LP6005441-DNA_B10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Papuan_HGDP00551-M-LP6005443-DNA_F08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Papuan_HGDP00551-M-LP6005443-DNA_F08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Papuan_HGDP00551-M-LP6005443-DNA_F08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Papuan_HGDP00552-F-LP6005443-DNA_D08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Papuan_HGDP00552-F-LP6005443-DNA_D08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Papuan_HGDP00552-F-LP6005443-DNA_D08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Papuan_HGDP00553-M-LP6005443-DNA_B08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Papuan_HGDP00553-M-LP6005443-DNA_B08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Papuan_HGDP00553-M-LP6005443-DNA_B08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Papuan_HGDP00554-F-LP6005443-DNA_E08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Papuan_HGDP00554-F-LP6005443-DNA_E08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Papuan_HGDP00554-F-LP6005443-DNA_E08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Papuan_HGDP00555-M-LP6005443-DNA_C07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Papuan_HGDP00555-M-LP6005443-DNA_C07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Papuan_HGDP00555-M-LP6005443-DNA_C07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Papuan_HGDP00556-M-LP6005443-DNA_G07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Papuan_HGDP00556-M-LP6005443-DNA_G07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Papuan_HGDP00556-M-LP6005443-DNA_G07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Pathan_HGDP00216-M-LP6005441-DNA_C10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Pathan_HGDP00216-M-LP6005441-DNA_C10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Pathan_HGDP00216-M-LP6005441-DNA_C10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Pathan_HGDP00232-F-LP6005441-DNA_D10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Pathan_HGDP00232-F-LP6005441-DNA_D10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Pathan_HGDP00232-F-LP6005441-DNA_D10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Piapoco_HGDP00702-F-LP6005441-DNA_A04 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Piapoco_HGDP00702-F-LP6005441-DNA_A04 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Piapoco_HGDP00702-F-LP6005441-DNA_A04
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Piapoco_HGDP00706-F-LP6005441-DNA_B04 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Piapoco_HGDP00706-F-LP6005441-DNA_B04 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Piapoco_HGDP00706-F-LP6005441-DNA_B04
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Pima_HGDP01044-F-LP6005441-DNA_F10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Pima_HGDP01044-F-LP6005441-DNA_F10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Pima_HGDP01044-F-LP6005441-DNA_F10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Pima_HGDP01047-M-LP6005441-DNA_E10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Pima_HGDP01047-M-LP6005441-DNA_E10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Pima_HGDP01047-M-LP6005441-DNA_E10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_PJL_Punjabi_Lahore_HG02783-M-LP6005442-DNA_A12 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_PJL_Punjabi_Lahore_HG02783-M-LP6005442-DNA_A12 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_PJL_Punjabi_Lahore_HG02783-M-LP6005442-DNA_A12
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_PJL_Punjabi_Lahore_HG02790-F-LP6005442-DNA_B12 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_PJL_Punjabi_Lahore_HG02790-F-LP6005442-DNA_B12 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_PJL_Punjabi_Lahore_HG02790-F-LP6005442-DNA_B12
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Russian_HGDP00887-M-LP6005441-DNA_G10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Russian_HGDP00887-M-LP6005441-DNA_G10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Russian_HGDP00887-M-LP6005441-DNA_G10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Russian_HGDP00903-F-LP6005441-DNA_H10 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Russian_HGDP00903-F-LP6005441-DNA_H10 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Russian_HGDP00903-F-LP6005441-DNA_H10
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_San_HGDP00987-M-LP6005441-DNA_B11 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_San_HGDP00987-M-LP6005441-DNA_B11 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_San_HGDP00987-M-LP6005441-DNA_B11
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_San_HGDP00991-M-LP6005443-DNA_G08 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_San_HGDP00991-M-LP6005443-DNA_G08 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_San_HGDP00991-M-LP6005443-DNA_G08
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_San_HGDP01032-M-LP6005441-DNA_A11 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_San_HGDP01032-M-LP6005441-DNA_A11 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_San_HGDP01032-M-LP6005441-DNA_A11
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Sardinian_HGDP01078-F-LP6005441-DNA_D11 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Sardinian_HGDP01078-F-LP6005441-DNA_D11 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Sardinian_HGDP01078-F-LP6005441-DNA_D11
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Sardinian_HGDP01079-M-LP6005441-DNA_C11 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Sardinian_HGDP01079-M-LP6005441-DNA_C11 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Sardinian_HGDP01079-M-LP6005441-DNA_C11
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_She_HGDP01333-M-LP6005443-DNA_F01 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_She_HGDP01333-M-LP6005443-DNA_F01 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_She_HGDP01333-M-LP6005443-DNA_F01
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_She_HGDP01335-F-LP6005443-DNA_G01 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_She_HGDP01335-F-LP6005443-DNA_G01 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_She_HGDP01335-F-LP6005443-DNA_G01
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Sindhi_HGDP00195-F-LP6005441-DNA_H11 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Sindhi_HGDP00195-F-LP6005441-DNA_H11 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Sindhi_HGDP00195-F-LP6005441-DNA_H11
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Sindhi_HGDP00208-M-LP6005441-DNA_G11 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Sindhi_HGDP00208-M-LP6005441-DNA_G11 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Sindhi_HGDP00208-M-LP6005441-DNA_G11
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Somali_Ayodo_81S-F-LP6005442-DNA_D09 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Somali_Ayodo_81S-F-LP6005442-DNA_D09 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Somali_Ayodo_81S-F-LP6005442-DNA_D09
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Surui_HGDP00846-F-LP6005441-DNA_A12 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Surui_HGDP00846-F-LP6005441-DNA_A12 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Surui_HGDP00846-F-LP6005441-DNA_A12
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Surui_HGDP00852-F-LP6005441-DNA_B12 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Surui_HGDP00852-F-LP6005441-DNA_B12 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Surui_HGDP00852-F-LP6005441-DNA_B12
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Thai_CHI_007-M-LP6005443-DNA_A07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Thai_CHI_007-M-LP6005443-DNA_A07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Thai_CHI_007-M-LP6005443-DNA_A07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Thai_CHI_034-F-LP6005443-DNA_B07 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Thai_CHI_034-F-LP6005443-DNA_B07 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Thai_CHI_034-F-LP6005443-DNA_B07
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Tlingit_Ale14-F-LP6005443-DNA_D05 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Tlingit_Ale14-F-LP6005443-DNA_D05 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Tlingit_Ale14-F-LP6005443-DNA_D05
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Tlingit_Ale32-F-LP6005443-DNA_E05 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Tlingit_Ale32-F-LP6005443-DNA_E05 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Tlingit_Ale32-F-LP6005443-DNA_E05
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Tubalar_Tuba19-F-LP6005442-DNA_F12 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Tubalar_Tuba19-F-LP6005442-DNA_F12 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Tubalar_Tuba19-F-LP6005442-DNA_F12
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Tubalar_Tuba9-F-LP6005442-DNA_E12 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Tubalar_Tuba9-F-LP6005442-DNA_E12 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Tubalar_Tuba9-F-LP6005442-DNA_E12
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Tu_HGDP01350-M-LP6005443-DNA_H01 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Tu_HGDP01350-M-LP6005443-DNA_H01 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Tu_HGDP01350-M-LP6005443-DNA_H01
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Tu_HGDP01355-F-LP6005441-DNA_D12 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Tu_HGDP01355-F-LP6005441-DNA_D12 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Tu_HGDP01355-F-LP6005441-DNA_D12
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Tujia_HGDP01095-M-LP6005443-DNA_A02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Tujia_HGDP01095-M-LP6005443-DNA_A02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Tujia_HGDP01095-M-LP6005443-DNA_A02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Tujia_HGDP01098-F-LP6005441-DNA_F12 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Tujia_HGDP01098-F-LP6005441-DNA_F12 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Tujia_HGDP01098-F-LP6005441-DNA_F12
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Tuscan_HGDP01163-M-LP6005441-DNA_G12 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Tuscan_HGDP01163-M-LP6005441-DNA_G12 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Tuscan_HGDP01163-M-LP6005441-DNA_G12
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Tuscan_HGDP01168-F-LP6005441-DNA_H12 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Tuscan_HGDP01168-F-LP6005441-DNA_H12 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Tuscan_HGDP01168-F-LP6005441-DNA_H12
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Ulchi_Ul31-F-LP6005442-DNA_H12 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Ulchi_Ul31-F-LP6005442-DNA_H12 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Ulchi_Ul31-F-LP6005442-DNA_H12
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Ulchi_Ul5-F-LP6005442-DNA_G12 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Ulchi_Ul5-F-LP6005442-DNA_G12 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Ulchi_Ul5-F-LP6005442-DNA_G12
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Uygur_HGDP01297-M-LP6005443-DNA_B02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Uygur_HGDP01297-M-LP6005443-DNA_B02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Uygur_HGDP01297-M-LP6005443-DNA_B02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Uygur_HGDP01306-F-LP6005442-DNA_B01 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Uygur_HGDP01306-F-LP6005442-DNA_B01 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Uygur_HGDP01306-F-LP6005442-DNA_B01
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Xibo_HGDP01246-M-LP6005443-DNA_C02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Xibo_HGDP01246-M-LP6005443-DNA_C02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Xibo_HGDP01246-M-LP6005443-DNA_C02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Xibo_HGDP01250-M-LP6005442-DNA_D01 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Xibo_HGDP01250-M-LP6005442-DNA_D01 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Xibo_HGDP01250-M-LP6005442-DNA_D01
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Yakut_HGDP00951-M-LP6005443-DNA_D02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Yakut_HGDP00951-M-LP6005443-DNA_D02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Yakut_HGDP00951-M-LP6005443-DNA_D02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Yakut_HGDP00956-F-LP6005442-DNA_F01 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Yakut_HGDP00956-F-LP6005442-DNA_F01 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Yakut_HGDP00956-F-LP6005442-DNA_F01
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Yi_HGDP01179-M-LP6005442-DNA_G01 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Yi_HGDP01179-M-LP6005442-DNA_G01 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Yi_HGDP01179-M-LP6005442-DNA_G01
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Yi_HGDP01188-F-LP6005442-DNA_H01 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Yi_HGDP01188-F-LP6005442-DNA_H01 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Yi_HGDP01188-F-LP6005442-DNA_H01
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Yoruba_HGDP00928-F-LP6005442-DNA_B02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Yoruba_HGDP00928-F-LP6005442-DNA_B02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Yoruba_HGDP00928-F-LP6005442-DNA_B02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Yoruba_HGDP00932-M-LP6005442-DNA_A02 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Yoruba_HGDP00932-M-LP6005442-DNA_A02 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Yoruba_HGDP00932-M-LP6005442-DNA_A02
python assess_variance_over_DTS.py --GC_tabix /net/eichler/vol7/home/psudmant/genomes/GC_tracks/hg19-gc-w200.GC_content.bed_sorted.gz --dup_tabix /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups_sorted.gz --contigs /net/eichler/vol7/home/psudmant/EEE_Lab/1000G/1000genomesScripts/windowed_analysis/DTS_window_analysis/windows/hg19_slide/500_bp_windows.pkl.contigs --in_DTS /net/eichler/vol19/projects/human_population_sequencing/nobackups/psudmant/wnd_output_DTS/hg19/C_team/renamed/500_bp_Zapotec_zapo0098-M-LP6005443-DNA_A12 --gaps /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/gaps/gap.bed.gz --segdups /net/eichler/vol7/home/psudmant/genomes/annotations/hg19/superdups/superdups.merged.bed.gz --window_size 500 --GC_DTS /net/eichler/vol7/home/psudmant/genomes/GC_tracks/windowed_DTS/HG19/HG19_slide500_bp.GC_content --out_total /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/total/500_bp_Zapotec_zapo0098-M-LP6005443-DNA_A12 --out_by_GC /net/eichler/vol7/home/psudmant/public_html/WorldWide_SD_diversity/QC/depth_cvg_var/by_GC/500_bp_Zapotec_zapo0098-M-LP6005443-DNA_A12
